src/components/user/user.service.ts

```diff
@@ -293,6 +293,12 @@ export class UserService {
             canReadCountry: true,
             canCreateProjectMember: true,
             canReadProjectMembers: true,
+            canCreateLanguageEngagement: true,
+            canReadLanguageEngagements: true,
+            canCreateInternshipEngagement: true,
+            canReadInternshipEngagements: true,
```

Comment:
I believe these could be combined to just `canCreateEngagement`, `canReadEngagements`